<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleName">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="CViewerContainer" name="viewer_container" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QDockWidget" name="setup_dockwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>382</width>
     <height>193</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>382</width>
     <height>193</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Setup</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="setup_dockwidget_contents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="sensors_label">
         <property name="text">
          <string>Sensor Combination:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="sensors_cbox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RGB-D - RGB-D</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="irx_label">
         <property name="text">
          <string>iRx:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="irx_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="accessibleName">
          <string>irx</string>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="iry_label">
         <property name="text">
          <string>iRy:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="iry_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="accessibleName">
          <string>iry</string>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="irz_label">
         <property name="text">
          <string>iRz:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QDoubleSpinBox" name="irz_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="accessibleName">
          <string>irz</string>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="itx_label">
         <property name="text">
          <string>iTx:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="itx_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="accessibleName">
          <string>itx</string>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="ity_label">
         <property name="text">
          <string>iTy:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="ity_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="accessibleName">
          <string>ity</string>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="itz_label">
         <property name="text">
          <string>iTz:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QDoubleSpinBox" name="itz_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="accessibleName">
          <string>itz</string>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="med_label">
         <property name="text">
          <string>MED:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="med_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="load_rlog_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load Rawlog</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="observations_dockwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Observations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="observations_dockwidget_contents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTreeView" name="observations_treeview">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="config_dockwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>382</width>
     <height>79</height>
    </size>
   </property>
   <property name="accessibleName">
    <string>config_dock</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Algorithm Configuration</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="config_dockwidget_contents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>382</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="accessibleName">
     <string>config_content</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="algo_group">
       <item>
        <widget class="QLabel" name="algo_label">
         <property name="text">
          <string>Calibration Algorithm:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="algo_cbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Plane matching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Line Matching</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CViewerContainer</class>
   <extends>QWidget</extends>
   <header location="global">viewer/CViewerContainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
